schemas/draft-07/v1.0.0/collection/request.json

```diff
@@ -266,6 +266,14 @@
             ],
             "description": "The time taken by this particular HTTP transaction to complete is stored in this field. For manually created responses, this field can be set to ``null``."
           },
+          "timings": {
+            "title": "Response Timings",
+            "type": [
+              "object",
+              "null"
+            ],
+            "description": "Set of timing information related to request and response in milliseconds"
```

Comment:
Keep the title and description together as has been done in the rest of the changes above.